### PR TITLE
Create Europe Flags.style

### PR DESCRIPTION
**Pull request by REEMSHANAB**

### CODE REVIEW

File: Europe Flags.style [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<img src="https://www.countries-ofthe-world.com/flags-normal/flag-of-Albania.png" width="500" height="300">
```
Comment (iarobinson) [inline Europe Flags.style:1]: Could we just have the URLs entered into the JavaScript object. For example:

```
const theEarthObjectJS = {
  continents: {
    africa: {
      "wikipedia_link": "https://en.wikipedia.org/wiki/Africa",
      "nations": {
        "angola": {
          "flag": "https://www.countries-ofthe-world.com/flags-normal/flag-of-Albania.png"
        }
      }
    },
```

**Review by iarobinson — CHANGES_REQUESTED** [review]
Thanks for the contribution REEMSHANAB! We want to add those image URLs to the javascript object. Please format it that way if you want the change to be approved.

File: Europe Flags.style (hunk 1 above)
Comment (REEMSHANAB) [reply Europe Flags.style:1]: Did I update the error correctly? Or am I still missing something?

Comment (REEMSHANAB) [reply Europe Flags.style:1]: @iarobinson

**Comment by iarobinson** [comment]
@REEMSHANAB - Just pinging you again about this one. Please reopen if you'd like to working this. :D Thanks for the effort.

**Comment by REEMSHANAB** [comment]
I am still confused, is my pull request inadequate for the code source?

**Pull request closed without merging**